docs/enterprise.md

```diff
@@ -0,0 +1,104 @@
+# Logfire Enterprise
+
+## Overview
+
+In addition to the [Pro plan](https://pydantic.dev/pricing), Logfire has two enterprise offerings:
+
+- **Enterprise Cloud**: Fully managed, SLA-backed service
+- **Enterprise Self-Hosted**: On-premises deployment via Kubernetes
+
+> **Note:** The Helm chart for Pydantic Logfire is now [open source](https://github.com/pydantic/logfire-helm-chart)
+
+## Enterprise Cloud
+
+### Target Users
+
+Organizations requiring:
+
+- Service Level Agreements (SLAs)
+- Custom Data Processing Agreements (DPAs)
+- HIPAA Business Associate Agreements (BAAs)
+- Invoice Billing
+- Custom retention
+
+### Features
+
+| Feature | Description |
+|---------|-------------|
+| **Full-Service Hosting** | All operational complexity managed by our engineering team |

```

Comment:
These seem like they could just be bullet points with bolded labels, not a table

Reply:
tables look better